### ISSUE TITLE
angular2-prototype issues

### ISSUE THREAD

**Issue by williamyinssw** [SharePoint/sp-dev-fx-webparts#111]
#### Category
- [ ] Question
- [x] Bug
- [x] Enhancement

#### Expected or Desired Behavior
This sample works fine locally, when I deploy it to SharePoint online, I was expecting to see the similar result.

#### Observed Behavior
but 
1) I saw the below errors:
[SPWebPartErrorCode.ScriptLoadError]:: Unable to load web part WebPart.null.g_932c30c6_be8d_4499_ac39_a52db655cde8,Error: ***Failed to load component "244f3542-af19-46e1-bd15-d6b9a35b9960" (null). Original error: Error loading https://component-id.invalid/244f3542-af19-46e1-bd15-d6b9a35b9960_0.0.1 script resources due to: undefined. CALLSTACK:: Error at t [as constructor] (https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-loader_en-us_4ec28636b0f9b465bd02537852aebf16.js:57:16731) at new t (https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-webpart-base_en-us_8d0a284d4d0d8e404879f4b5fcc7cc99.js:47:22096) at Function.t.create (https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-webpart-base_en-us_8d0a284d4d0d8e404879f4b5fcc7cc99.js:47:22509) at https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-webpart-base_en-us_8d0a284d4d0d8e404879f4b5fcc7cc99.js:48:627

2) Also, I noticed the output js file is called "hello-world.bundle.js", it should be called a more proper name. (I know I can change it by myself, but it would be nicer it can be fixed from the original place)

3) I saw the output js file is 1.6Mb, is it possible to reduce the size?

#### Steps to Reproduce
Basically, I just followed the steps to deploy to SharePoint online:
1. gulp package-solution
2. upload the output package into app category.  (developer tenant)
3. add that in a test SharePoint site collection instance
4. add a page, then add the web part into that page, then I see the errors.

**Comment by chakkaradeep** [comment]
Two things:
1) I dont believe the prototype is updated for RC0 and hence you are seeing it fail when uploaded to SharePoint.

2) We are still working on this prototype. At its current state, it will work in local as its referencing the Drop6 builds.

We will be updating the prototype soon. 

Thanks!

**Comment by webtechy** [comment]
Ensure you have specified a CDN location (config/write-manifests.json) for your assets (or run "gulp serve" and let it use localhost for testing). To ensure this value is used, you have to run:

```
gulp clean
gulp build
gulp bundle --ship
gulp package-solution --ship
```

All steps above very important (clean removes cached versions that don't reference the CDN, the --ship makes sure the CDN is put into the manifests).
